modules/monitoring-understanding-the-cluster-observability-operator.adoc

```diff
@@ -0,0 +1,32 @@
+//Module included in the following assemblies:
+//
+// monitoring/cluster_observability_operator/cluster-observability-operator-overview.adoc
+
+:_content-type: CONCEPT
+[id="understanding-the-cluster-observability-operator_{context}"]
+= Understanding the Cluster Observability Operator
+
+A default monitoring stack created by the Cluster Observability Operator (COO) includes a highly available Prometheus instance capable of sending metrics to an external endpoint by using remote write.
+
+Each COO stack also includes an optional Thanos Querier component, which you can use to query a highly available Prometheus instance from a central location, and an optional Alertmanager component, which you can use to set up alert configurations for different services.
```

Comment:
Could possibly split this into a "... includes the following components:" style and list the 2 as bullets, if you like. Just to break up the kinda-long sentence.

Reply:
Yeah, I hear you, but a bulleted list of components appears in the previous section, so I think I'll leave this one as-is so as not to be repetitive, even though I agree that the sentence is bit longer than I would like.